README.md

```diff
@@ -0,0 +1,68 @@
+# ScyllaDB JS Driver
+
+## Instalation and build process
+
+For instalation type command below:
+
+```
+$ npm install
+```
+
+For build process use this command:
+
+```
+$ npm run build
+```
+
+If command above don't work, make sure that you have napi package downloded globally.
+
+## File structure
+
+| File / Folder     | Origin                             | Usage                                  | Shouldn't be in .gitignore |
+| ----------------- | ---------------------------------- | -------------------------------------- | -------------------------- |
+| .cargo            | Generated by 'napi-rs'             | ?                                      | ?                          |
+| .github           | Generated by 'napi-rs'             | CI-CD                                  | ✅                         |
```

Comment:
In general, I really like that you created this table. I helps a lot to understand the project's structure.